cmd/update/matchable_execution_cluster_label.go

```diff
@@ -0,0 +1,76 @@
+package update
+
+import (
+	"context"
+	"fmt"
+
+	sconfig "github.com/flyteorg/flytectl/cmd/config/subcommand"
+	"github.com/flyteorg/flytectl/cmd/config/subcommand/executionclusterlabel"
+	cmdCore "github.com/flyteorg/flytectl/cmd/core"
+)
+
+const (
+	executionClusterLabelShort = "Updates matchable resources of execution cluster label"
+	executionClusterLabelLong  = `
+Updates execution cluster label for given project and domain combination or additionally with workflow name.
+
+Updating to the execution cluster label is only available from a generated file. See the get section for generating this file.
+Also this will completely overwrite any existing custom label on project and domain and workflow combination.
+Would be preferable to do get and generate an attribute file if there is an existing attribute already set and then update it to have new values
```

Comment:
is this advice necessary seeing as there can only be one label at a time?

Reply:
Yeah not required. removed it.